beak.swift

```diff
@@ -46,7 +46,7 @@ public func link() throws {
 
     for bashConfigFile in [".bash_profile", ".bashrc", ".profile"] {
```

Comment:
You forgot to add the zsh configuration file to the array here. It should be created if not already existent.